src/test/components/ThreadActivityGraph.test.js

```diff
@@ -200,99 +204,93 @@ describe('ThreadActivityGraph', function() {
     expect(flushDrawLog()).toMatchSnapshot();
   });
 
-  /**
-   * The ThreadActivityGraph is not a connected component. It's easiest to test it
-   * as once it's connected to the Redux store in the SelectedActivityGraph.
-   */
-  describe('ThreadActivityGraph', function() {
```

Comment:
This part is only that I removed this extra describe block and moved everything to the outer block. You can double check by unchecking whitespace changes.